lib/CodeGen/CGDynamicCheck.cpp

```diff
@@ -0,0 +1,68 @@
+//===---- CGDynamicCheck.cpp - Emit LLVM Code for Checked C Dynamic Checks -===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+//
+// This contains code to emit Checked C Dynamic Checks as LLVM code.
+//
+//===----------------------------------------------------------------------===//
+
+#include "CodeGenFunction.h"
+#include "llvm/ADT/Statistic.h"
+
+using namespace clang;
+using namespace CodeGen;
+using namespace llvm;
+
+#define DEBUG_TYPE "DynamicCheckCodeGen"
+
+STATISTIC(DynamicChecksFound, "Number of Dynamic Checks Found");
```

Comment:
Capitalization of the names of statistics may be inconsistent with capitalization for other statistics.   I don't think other names are mostly in upper case.